lib/util/check-publish.js

```diff
@@ -26,6 +26,12 @@ exports.checkPublish = function checkPublish(context, filePath, targets) {
         return
     }
 
+    // Private packages are never published so we don't need to check the imported dependencies either.
+    // More information: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#private
+    if (packageInfo.private) {
```

Comment:
better to change the check to be more correctly to avoid false negatives (`"private": false`).
```suggestion
    if (packageInfo.private === true) {
```

Reply:
I updated the check as suggested 👍.